linux/compat.bsh

```diff
@@ -150,7 +150,39 @@ function load_vsi_compat()
   # ====
   #
   # The following variables will help cope with the difference in bash versions as seamlessly as possible.
+  #**
+
+  # Make life easier (and quicker after 6 uses)
+  local bash_major_version="${BASH_VERSINFO[0]}"
+  local bash_minor_version="${bash_major_version}${BASH_VERSINFO[1]}"
+
+  #**
+  # .. var:: bash_feature_posix_process_substitution
+  #
+  # In bash 5.0 and older, process substitution isn't allows when in posix mode. This issue can arise if a function contains non-posix commands and is exported, before calling ``bash --posix``. Bash will complain about each and every function that is non-posix.
+  #
+  # :Value: * ``0`` - Bash does support parameter transformations (Bash 4.4 or newer)
```

Comment:
Here you say 4.4 of newer, but in the summary, you said 5.0 or older

Reply:
It also says "parameter transformation", bad copy and paste I suspect